Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -15,10 +15,27 @@
 
 @implementation BPPacker
 
++ (NSMutableSet *)fullyExpandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
```

Comment:
This function is a bit confusing and folks may forget to call for both testToInclude and testToSkip. Could you refactor this method to take in a BPConfiguration file and spit our another BPConfiguration with a list fully qualified tests?

like `santinizeTestNameInBPConfiguration:withTestFile:`